taxonomies/additives.txt

```diff
@@ -22235,7 +22236,7 @@ additives_classes:en: en:antioxidant
 vegan:en:yes
 vegetarian:en:yes
 
-en:E1103, Invertase
+en:E1103, Invertase, E-1103, E 1103, INS1103, INS-1103, INS 1103, SIN1103, SIN-1103, SIN 1103
```

Comment:
All the variants of E numbers in additives names are not needed as synonyms, they are handled during parsing.

Reply:
Thanks. It is removed now

Reply:
Thank you!